README.md

```diff
@@ -32,18 +36,25 @@ export LOADTESTING_API_KEY="FIRETEXT_SIMULATION_KEY"
 export FIRETEXT_API_KEY="FIRETEXT_ACTUAL_KEY"
 export STATSD_PREFIX="FAKE_PREFIX"
 export NOTIFICATION_QUEUE_PREFIX="PREFIX-TO-IDENTIFY-SQS-QUEUE"
+export AWS_REGION=<YOUR AWS REGION>
```

Comment:
I had that originally, but thought maybe I should generalise. Happy to hard code though :+1: